### PR TITLE
update readme, reorganize React components, remove unused packages

### PR DESCRIPTION
**Pull request by eyang4**
### Assignee Tasks

* [ ] added unit tests (or none needed)
* [ ] written relevant docs (or none needed)
* [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

_Your PR Notes Here_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,11 +50,9 @@
     "css-loader": "^3.4.2",
     "d3": "^5.15.0",
     "d3-geo": "^1.11.9",
-    "d3-svg-legend": "^2.25.6",
     "express": "^4.16.4",
     "express-session": "^1.15.1",
     "history": "^4.9.0",
-    "leaflet": "^1.6.0",
```
Comment (zkarmi) [inline package.json]: I don't want to remove leaflet -- my next goal after tooltips will be to implement the base layers and overlays, which I'll need leaflet for.

Comment (zkarmi) [reply package.json]: I included it in my highlights below, but removing `react-d3-library` and `react-leaflet-d3-svg-overlay` is fine by me.

File: public/index.html [hunk 2]
```diff
@@ -5,12 +5,6 @@
     <meta name='viewport' content='width=device-width initial-scale=1.0'>
     <title>Delve.NYC</title>
     <link rel="stylesheet" href="/style.css" />
-    <link
```
Comment (zkarmi) [inline public/index.html]: As in package.json, I'd like to keep leaflet for next steps with map layering.

File: package.json (hunk 1 above)
Comment (eyang4) [reply package.json]: Works for me

**Pull request merged**